repairnator/src/main/java/fr/inria/spirals/repairnator/process/step/CheckoutSourceCodeForPreviousBuild.java

```diff
@@ -0,0 +1,47 @@
+package fr.inria.spirals.repairnator.process.step;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.eclipse.jgit.api.Git;
+import org.eclipse.jgit.api.errors.GitAPIException;
+
+import fr.inria.spirals.repairnator.process.ProjectState;
+import fr.inria.spirals.repairnator.process.inspectors.ProjectInspector;
+
+/**
+ * Created by fermadeiral.
+ */
+public class CheckoutSourceCodeForPreviousBuild extends CloneRepository {
+
```

Comment:
Ok we're having a vocabulary issue there.
What's the difference between checking out the build and checking out the source code of the build? You can only checkout the code no?